cvat-core/src/session.js

```diff
@@ -1406,9 +1417,234 @@
         }
     }
 
+    /**
+     * Class representing a project
+     * @memberof module:API.cvat.classes
+     * @extends Session
+     */
+    class Project extends Session {
```

Comment:
Session is supposed to be have quite wide API:  annotations, actions, frames, logger, etc.
Project doesn't provide it, so, I think we should not extends the Session class here

Reply:
Sure, moved to separate file